src/components/layout/header/header.tsx

```diff
@@ -123,8 +135,20 @@ export const Header = () => {
   }, [isUserUnauthorized, isCartEmpty, toggleAuthStatus, location.pathname]);
 
   useEffect(() => {
+    const fetchCartData = async () => {
+      try {
+        const cartData = await apiGetCartById();
+        if (cartData) {
+          setCart(cartData);
+        }
+      } catch {
```

Comment:
you catch, but do not display the problem with the basket
```suggestion
useEffect(() => {
  const fetchCartData = async () => {
    try {
      const cartData = await apiGetCartById();
      setCart(cartData ?? null);
    } catch () {
     // view 'Failed to fetch cart:'
      setCart(null);
    }
  };

  fetchCartData();
  updateNavItems();
}, [updateNavItems]);

```

Reply:
added handleCatchError(error, 'Error getting cart data');